srcjs/bindings/sliderInputBinding.js

```diff
@@ -47,19 +46,16 @@ $.extend(f7SliderBinding, {
 
   // Given the DOM element for the input, return the value
   getValue: function(el) {
-    this.app = getAppInstance();
     return this.app.range.getValue($(el)[0]);
```

Comment:
`el` should also work instead of `$(el)[0]`

Reply:
I'm taking a closer look at the range slider now, it doesn't work in all cases yet- I'll look into this as well 👍